jbmc/regression/jbmc/dereference-virtual-call/Test.java

```diff
@@ -0,0 +1,26 @@
+public class Test {
+
+  public static void main(int argc) {
```

Comment:
:pick: Would rather only use main methods for actual main methods (e.g. one that takes a `String[] args`)